draft-ietf-httpbis-priority.md

```diff
@@ -471,9 +473,9 @@ PRIORITY_UPDATE on wrong stream, a PRIORITY_UPDATE with an invalid ID, etc.
 It is not always the case that the client has the best understanding of how the
 HTTP responses deserve to be prioritized. For example, use of an HTML document
 might depend heavily on one of the inline images. Existence of such
-dependencies is typically best known to the server.
+dependencies is typically best known to the origin server.
 
-By using the "Priority" response header, a server can override the
+By using the "Priority" response header, an origin server can override the
```

Comment:
I am not sure if we should use "origin" here. Assuming that we would not forbid an intermediary from setting the priority header field, I think just saying "server" (or maybe "upstream") might be sufficient.

The same goes to the proposed change from "client" to "user agent."

Reply:
> we would not forbid an intermediary from setting the priority header field
I agree.

I interpreted the end-to-end section as "UA -> Origin Server" https://github.com/kazuho/http-extensions/blob/master/draft-ietf-httpbis-priority.md#why-use-an-end-to-end-header-field

:)